dotnet-mfa-example/WorkOS_MFA_Example_App.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>

```

Comment:
Could call out in the Readme that if using M1 and Monterrey that you need to adjust the TargetFramework to 6.0